CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## DEV
+
+* Increased maximum allowed version of Cryptography library
```

Comment:
I think it will make sense to specify here the version of cryptography our package requires.